### PR TITLE
README and Code of Conduct additions

### PR DESCRIPTION
**Pull request by DrKJeff16**
I'm editing those files to include more detailed and easy-to-read information. Will keep making updates, and check any suggestions made.

### CODE REVIEW

**Comment by DrKJeff16** [comment]
The drone integration escapes my power:(

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,5 +1,25 @@
 # Contributor Covenant Code of Conduct
 
+* **Last Updated:** 02/07/2021
```
Comment (JustinBack) [inline CODE_OF_CONDUCT.md:3]: Not really necessary? Git shows commits

**Review by JustinBack — CHANGES_REQUESTED** [review]
I like, tho the last changed date is not necessary due to git

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (DrKJeff16) [reply CODE_OF_CONDUCT.md]: A bit automatic, though understandable. Deleting rn.

**Review by JustinBack — APPROVED** [review]

**Pull request merged**